board/safety/safety_gm.h

```diff
@@ -85,19 +86,16 @@ static int gm_rx_hook(CANPacket_t *to_push) {
     if (addr == 241) {
       // Brake pedal's potentiometer returns near-zero reading
       // even when pedal is not pressed
-      brake_pressed = GET_BYTE(to_push, 1) >= 10U;
+      brake_pressed = (GET_BYTE(to_push, 1) >= 10U) || (gm_regen_braking != 0);
```

Comment:
In test_models we do a brake_pressed comparison, and in GM's CarState it ORs brake and regen together. We were unlucky to never test any segments with regen braking, so we never hit a mismatch.